index.coffee

```diff
@@ -1,157 +1,173 @@
 ChildProcess = require 'child_process'
-fs = require 'fs'
 asar = require 'asar'
-path = require 'path'
 temp = require 'temp'
 _ = require 'underscore'
+jetpack = require 'fs-jetpack'
 
 temp.track()
 
-module.exports = (grunt) ->
+consoleLogger =
+  error: (args...) ->
+    console.error.apply(null, args)
+  warn: (args...) ->
+    console.warn.apply(null, args)
+  info: (args...) ->
+    console.info.apply(null, args)
+  debug: ->
+
+nullLogger =
+  error: ->
+  warn: ->
+  info: ->
+  debug: ->
+
+module.exports.build = (config, done) ->
+  log = config.log || config.log == false ? nullLogger : consoleLogger
```

Comment:
CoffeeScript does not have the ternary operator `a ? b : c`

This could be:

```coffee
if config.log is false
  log = nullLogger
else
  log = config.log ? consoleLogger
```